content/pages/features.html

```diff
@@ -1,6 +1,7 @@
 <html>
   <head>
     <meta name="title" content="Product Features" />
+    <meta name="description" content="Read the Docs includes docs as code features such as building docs, hosting them, search, previewing proposed PR changes, and authentication for private docs." />
```

Comment:
I usually point out `docs` -> `documentation` to disambiguate between legal/word/technical "docs".

```suggestion
    <meta name="description" content="Read the Docs includes docs as code features such as building and hosting documentation, search, previewing changes with pull request previews, and authentication for private documentation." />
```